node_app/client/index.html

```diff
@@ -201,7 +200,7 @@ <h4 class="interTitle">Where you can use it</h4>
         </div>
 
         <form>
-            <div id="formGenerate">
+            <div id="formGenerate2">
```

Comment:
`formGenerate` is duplicated; this might cause an error

Reply:
fixed :)